src/controllers/bookings.cr

```diff
@@ -1,3 +1,5 @@
+require "html"
```

Comment:
I don't think we need this require? Or is it for something else in the file?

Reply:
Sorry forgot to remove the require